### ISSUE TITLE
Lack of `extern crate proc_macro` causes geiger to fail

### ISSUE THREAD

**Issue by eliaslevy** [rust-secure-code/cargo-geiger#96]
Rust 1.42.0 finally did away with the need for a `extern crate proc_macro` statement when using the `proc_macro` crate in the 2018 edition.  Alas, removing this statement in a crate using `proc_macro` causes `cargo geiger` to fail:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Cargo("could not compile `test-macros`.")', src/libcore/result.rs:1165:5
```

The message in the output error JSON is `unresolved import "proc_macro"` and the code `E0432`.

**Comment by anderejd** [comment]
Thanks for the bug report. Can you try the latest version, I upgraded all dependencies yesterday and published 0.10.0.

If it still fails, can you provide a minimal reproducible example?

**Comment by eliaslevy** [comment]
Looks good.  thanks.
